sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/FileStreamSource.scala

```diff
@@ -41,36 +40,59 @@ class FileStreamSource(
     metadataPath: String,
     options: Map[String, String]) extends Source with Logging {
 
+  import FileStreamSource._
+
+  private val sourceOptions = new FileStreamOptions(options)
+
   private val fs = new Path(path).getFileSystem(sparkSession.sessionState.newHadoopConf())
   private val qualifiedBasePath = fs.makeQualified(new Path(path)) // can contains glob patterns
-  private val metadataLog = new HDFSMetadataLog[Seq[String]](sparkSession, metadataPath)
+
+  private val metadataLog = new HDFSMetadataLog[Seq[FileEntry]](sparkSession, metadataPath)
+
```

Comment:
I know it's bike-shedding, but, my personal taste is to keep declaration of member fields together into a block, or if there are many, perhaps logically group them. So just really depends on whether you mean metadataLog to stand a bit separately from other fields.